### DISCUSSION TITLE
v0.90.11-beta

### DISCUSSION THREAD

**Discussion by eliandoran** [TriliumNext/Trilium#6034] [Releases]
💡 Key highlights
-----------------

*   Now it is possible to have a fixed toolbar for editing text notes instead of the standard floating one, see [Classic editor for text notes (with fixed toolbar)](https://github.com/TriliumNext/Notes/pull/571) for more information.

🐞 Bugfixes
-----------

*   [Double input after spaces on Android](https://github.com/TriliumNext/Notes/issues/568)
*   [Hyperlinks with custom protocols not working](https://github.com/TriliumNext/Notes/issues/122) by @SiriusXT
*   [Internal Link: Two Tooltips are displayed](https://github.com/TriliumNext/Notes/issues/525)

✨ Improvements
--------------

*   [Insert inline images in text notes](https://github.com/TriliumNext/Notes/issues/531) (also grouped the image alignment buttons in subcategories since there were too many of them).
*   [Explicitly manage the "latest" tag, and have it point to the same tag as "stable"](https://github.com/TriliumNext/Notes/pull/545) by @perfectra1n
*   [Make attachments open in a new tab/browser](https://github.com/TriliumNext/Notes/pull/559) by @SiriusXT
*   [mouse scroll wheel direction for zoom level](https://github.com/TriliumNext/Notes/pull/555) by @rom1dep
*   Apply syntax highlight to preview of code notes as well.
*   [More reliably check for version updates](https://github.com/TriliumNext/Notes/pull/574) by @perfectra1n
*   README improvements by @perfectra1n

🌍 Internationalization
-----------------------

*   Preliminary support for the German language by @j13055 
*   Improved French translations by @Potjoe-97
*   Improved Spanish translations by @hasecilu

⬆️ Library updates
------------------

*   mind-elixir: 4.2.4 -> 4.3.1
*   client: force-graph: 1.45.0 -> 1.46.0

<hr /><em>This discussion was created from the release <a href='https://github.com/TriliumNext/Notes/releases/tag/v0.90.11-beta'>v0.90.11-beta</a>.</em>

**Comment by yu-jingrui** [comment]
Do I have to update the server if I just want to try out the windows desktop client?

**Reply by eliandoran** [reply]
@yu-jingrui , it should work just fine if you have a server that is at least 0.90.8.
